lib/retriable/exponential_backoff.rb

```diff
@@ -25,19 +25,16 @@ def initialize(opts = {})
 
     def intervals
       intervals = Array.new(tries) do |iteration|
-        [base_interval * multiplier**iteration, max_interval].min
+        [base_interval * multiplier ** iteration, max_interval].min
       end
 
-      return intervals if rand_factor.zero?
```

Comment:
the changes in this file are also just housekeeping changes - this line is unnecessary (i mean, it saves a single ruby call... but we are usually retrying operations that take at least seconds)